ccm_web/Dockerfile

```diff
@@ -23,13 +25,20 @@ FROM node:14-slim AS prod
 ENV NODE_ENV=production
 WORKDIR /app
 
-COPY --from=base /base/package.json /base/package-lock.json /base/start.sh ./
+COPY --from=base \
+    /base/package.json \
+    /base/package-lock.json \
+    /base/start.sh \
+    /base/ecosystem.config.js \
+    ./
 RUN npm install --production
-
+RUN npm install pm2@5.1.0 -g
```

Comment:
Is there a reason this is a separate step with a hardcoded version here in each of the Dockerfile's rather than just part of the package.json?

Reply:
Good question, not 100 percent certain I made the right choice. We don't use `pm2` in development. I guess if you do it this way, `package.json` doesn't prescribe a certain way to deploy your application, so you could write different `Dockerfile` and `start.sh` files to use an alternative if you wanted, adding some flexbility. ROHQ follows a similar approach actually, I'm realizing: https://github.com/tl-its-umich-edu/remote-office-hours-queue/blob/b076017a63f52e44dd2b5140d36ef1c2eeca8c9f/src/Dockerfile#L27-L29 Honestly, though I think the major reason I did it this way was `pm2` docs said to: https://pm2.keymetrics.io/docs/usage/docker-pm2-nodejs/ They're saying to install it globally in the container, so I guess I wasn't sure if I could do it in `package.json`. I could try if you think that would be better.

Reply:
I could also allow for patch upgrades here, I think. Pinning to some version range made sense to me though.

Reply:
@jonespm, any additional feedback here? Thinking about merging.